qutebrowser/utils/usertypes.py

```diff
@@ -297,11 +297,16 @@ def __init__(self, fileobj):
 
 class OpenFileDownloadTarget(DownloadTarget):
 
-    """Save the download in a temp dir and directly open it."""
+    """Save the download in a temp dir and directly open it.
 
-    def __init__(self):
+    Attributes:
+        cmdline: The command to use as string. A {} is expanded to the
```

Comment:
Please do - at some point I want to have autogenerated (Sphinx) documentation for all docstrings, as developer/API docs.